modules/UI/dialogs/InviteDialog.js

```diff
@@ -0,0 +1,352 @@
+/* global APP, $, JitsiMeetJS */
+
+import UIEvents from '../../../service/UI/UIEvents';
+
+/**
+ * Substate for password
+ * @type {{LOCKED: string, UNLOCKED: string}}
+ */
+const States = {
+    LOCKED: 'locked',
+    UNLOCKED: 'unlocked'
+};
+
+/**
+ * Class representing custom dialog
+ * for invitation of participants
+ * @class InviteDialog
+ */
+export default class InviteDialog {
+    constructor(options) {
+        let InviteAttributesKey = 'roomUrlDefaultMsg';
+        let title = APP.translation.translateString(InviteAttributesKey);
+
+        this.password = options.password;
+        this.roomUrl = options.roomUrl || null;
+        this.emitter = options.emitter || null;
```

Comment:
Changed to `APP.UI.addListener` and `APP.UI.emitEvent`. Does we use eventEmitter instance in UI object as a single event bus, right?